src/core/SampleBuffer.cpp

```diff
@@ -1482,3 +1484,7 @@ SampleBuffer::handleState::~handleState()
 {
 	src_delete( m_resamplingData );
 }
+
+
+
+//sample_t SampleBuffer::m_userAntiAliasWaveTable[Oscillator::WAVE_TABLES_PER_WAVEFORM_COUNT][Oscillator::WAVETABLE_LENGTH] = {0};
```

Comment:
I think we don't need to left this comment.

Reply:
Done.